### PR TITLE
Fetch: use .any.js syntax

### PR DESCRIPTION
**Pull request by annevk**

### CODE REVIEW

**Comment by annevk** [comment]
If folks are generally supportive of this idea I'll start adding all the other resources to this PR.

**Comment by w3c-bots** [comment]
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5129/)

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/210612159)

# Firefox (nightly channel) #
Testing web-platform-tests at revision 06e5b039e3a748ab51776f0425e3f6a856a6df24
Using browser at version BuildID 20170312110221; SourceStamp 3154da520af568d4da3de62bc26fbf97a1317f51
Starting 10 test iterations
No tests run.

**Comment by w3c-bots** [comment]
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/210612164)

# Chrome (unstable channel) #
Testing web-platform-tests at revision 06e5b039e3a748ab51776f0425e3f6a856a6df24
Using browser at version 58.0.3029.14 dev
Starting 10 test iterations
No tests run.

**Comment by jdm** [comment]
I am very much in favour of this.

File: fetch/api/basic/accept-header.any.js [hunk 1]
```diff
@@ -1,7 +1,4 @@
-if (this.document === undefined) {
-  importScripts("/resources/testharness.js");
-  importScripts("../resources/utils.js");
-}
+// META: script=../resources/utils.js
```
Comment (domenic) [inline fetch/api/basic/accept-header.any.js]: Woah, this works? That's neat...

**Comment by annevk** [comment]
I would prefer to get this landed and then cleanup more once #4102 is all done. @jdm could you review? (I did personally look through all results for .html and .worker.html auto-generated resources. I should probably do that again once it's all cleaned up as there are some browser bugs that need to be sorted somehow.)

**Comment by jdm** [comment]
Sorry about the delay; this caught me while I was on vacation.

**Review by jdm — APPROVED** [review]

**Comment by annevk** [comment]
Ah, thanks @jdm. I'll rebase first for the conflicting files.

**Comment by annevk** [comment]
I only modified fetch/api/cors/cors-preflight.any.js fetch/api/cors/cors-preflight-star.any.js while rebasing and it appears I successfully merged the changes made to those resources on master. So this can land once the checks are complete unless anyone has remaining concerns.

I'll be happy to land these myself and give them a clean commit message.

**Pull request merged**